shared/rakefiles/entrypoint.rake

```diff
@@ -177,8 +178,23 @@ task :destroy_tfstate, [:prefix] => [:set_vars, :check_destroy_allowed] do |task
   sh "docker volume rm -f -- #{ENV["TF_VAR_project_id"]}-#{ENV["USER"]}-terragrunt"
 end
 
+desc "[ADVANCED] Rotate Terraform state key for prefix, passed as argument -- rake rotate_tfstate_key['k8s']"
+task :rotate_tfstate_key, [:prefix] => [:set_vars, :check_destroy_allowed] do |taskname, args|
+  if args[:prefix].nil? || args[:prefix].size == 0
+    puts "Argument :prefix not present, defaulting to k8s"
+    prefix = "k8s"
+  else
+    prefix = args[:prefix]
+  end
+  sh "#{@exekube_cmd} rake rotate_secret['key_tfstate_encryption_key','default','sh -c \"gsutil \
+       -o GSUtil:decryption_key1=$TF_VAR_key_tfstate_encryption_key_rotated \
+       -o GSUtil:encryption_key=$TF_VAR_key_tfstate_encryption_key \
+       rewrite -k -r gs://#{ENV["TF_VAR_project_id"]}-tfstate/#{@env}/#{prefix}\"',skip_secret_mgmt,preserve_stderr]"
```

Comment:
Because rotating TF state key is just a special case of rotating a secret, and I decided to make `:rotate_secret` task universal. It rotates a secret by default, but when `:cmd` argument is present, it also executes command from that argument to handle cases when we need some additional action at the moment when we have both old and new secrets set as ENV vars.